pkg/utils/testing/testing.go

```diff
@@ -50,9 +50,25 @@ func NewPerformanceProfile(name string) *performancev1alpha1.PerformanceProfile
 			RealTimeKernel: &performancev1alpha1.RealTimeKernel{
 				Enabled: pointer.BoolPtr(true),
 			},
+			MachineConfigLabels: map[string]string{
```

Comment:
if we can determine defaults that make sense, we'll want unit tests that validate the defaults are used as expected when no values are set for some options.

Reply:
done